ksql-engine/src/test/java/io/confluent/ksql/function/udf/PluggableUdfTest.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2018 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.ksql.function.udf;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import io.confluent.ksql.security.ExtensionSecurityManager;
+
+public class PluggableUdfTest {
+
+  private final SecurityManager securityManager = System.getSecurityManager();
+  @Before
+  public void before() {
+    System.setSecurityManager(ExtensionSecurityManager.INSTANCE);
+  }
+
+  @After
+  public void after() {
+    System.setSecurityManager(securityManager);
+  }
+
+  @Test(expected = SecurityException.class)
+  public void shouldEnableUdfSecurity() {
```

Comment:
Hmm. This seems identical to the test case in `ExtensionSecurityManagerTest`. Any particular reason to duplicate?

Reply:
nope it is just where i first wrote it. Can remove